packages/theme/css/foundations/fade.css

```diff
@@ -8,7 +8,8 @@
   --uitk-color-grey-60-fade-foreground: rgba(197, 201, 208, var(--uitk-palette-opacity-foreground));
   --uitk-color-grey-70-fade-foreground: rgba(180, 183, 190, var(--uitk-palette-opacity-foreground));
   --uitk-color-grey-200-fade-foreground: rgba(97, 101, 110, var(--uitk-palette-opacity-foreground));
-  --uitk-color-grey-800-fade-foreground: rgba(36, 37, 38, var(--uitk-palette-opacity-foreground));
+  --uitk-color-grey-800-fade-foreground: rgba(132, 135, 142, var(--uitk-palette-opacity-foreground));
```

Comment:
@libertymayc This is the source of the regression. I tracked down the origin of this value and it seems to be for grey-90 not grey-800.